training/distributed_training/pytorch/data_parallel/roberta/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# ARG region
+
+# FROM 763104351884.dkr.ecr.${region}.amazonaws.com/pytorch-training:1.9.0-gpu-py38-cu111-ubuntu20.04
+
+# RUN mkdir /dependencies && cd /dependencies && git clone https://github.com/HerringForks/fairseq && cd fairseq && git checkout use_herring && pip install --upgrade pip && pip install --editable ./
+
+# RUN pip install numpy==1.20.1
+
+FROM 570106654206.dkr.ecr.us-west-2.amazonaws.com/diffusion-sagemaker-smddp:smddp-1.4.0-rc0
```

Comment:
Where is this image URI from? 
Would you explain more what's happening here? Why can't we use the DLC images?

Also this bounds the image to `us-west-2` only, and it results in limiting the whole example notebook to be usable only in `us-west-2`.